docs/config.md

```diff
@@ -0,0 +1,59 @@
+## Configuring Google Spreadsheet Operations
+
+To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element in your proxy configuration before use any other
```

Comment:
Mention where to add the element. 

To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element to your proxy configuration before using any other Google Spreadsheet connector operations.